cmscontrib/loaders/polygon.py

```diff
@@ -187,9 +187,13 @@ def get_task(self, get_statement=True):
             outfile_param = judging.attrib['output-file']
 
             checker_src = os.path.join(self.path, "files", "check.cpp")
+            if not os.path.exists(checker_src):
+                checker_src = os.path.join(self.path, "check.cpp")
+            checker_dir = os.path.dirname(self.path)
```

Comment:
Shouldn't this be `checker_dir = os.path.dirname(checker_src)` (or putting dirname() directly where you use it later)?